src/components/expanded-state/unique-token/saveToCameraRoll.js

```diff
@@ -0,0 +1,61 @@
+import CameraRoll from '@react-native-community/cameraroll';
+import { Alert, PermissionsAndroid, Platform } from 'react-native';
+import RNFetchBlob from 'rn-fetch-blob';
+
+const getPermissionAndroid = async () => {
+  try {
+    const granted = await PermissionsAndroid.request(
```

Comment:
@ibrahimtaveras00  make sure you test this on both android & iOS (Save to camera roll)

Reply:
broken on android atm